Core/Core/Extensions/UIViewControllerExtensions.swift

```diff
@@ -55,6 +55,16 @@ extension UIViewController {
         return false
     }
 
+    public var displayModeButtonItem: UIBarButtonItem? {
+        guard let splitView = splitViewController else { return nil }
+        let defaultButton = splitView.displayModeButtonItem
```

Comment:
Does the default appearance not look right so we need to use the custom icons?

Reply:
Yeah, and it doesn't have a "collapse" state. It's really weird. I don't like that we have to customize it but what we have makes sense. I tried really hard to stick to defaults but it's pretty different from what we have.

Reply:
sounds good.